Makefile

```diff
@@ -40,25 +40,14 @@ LDFLAGS  += 	-L$(XTENSA_LIBS)/lib \
 		-L$(XTENSA_LIBS)/arch/lib \
 
 
-CFLAGS+=-std=c99 -DESP8266
+CFLAGS+=-std=gnu99 -DESP8266 -DWITH_PGM_READ_HELPER
 
 CFLAGS += -Wall -Os -g -O2 -Wpointer-arith -Wl,-EL -nostdlib -mlongcalls -mno-text-section-literals  -D__ets__ -DICACHE_FLASH
 
 CFLAGS += -ffunction-sections -fdata-sections
 
 CFLAGS += -fdebug-prefix-map=$(PWD)= -fdebug-prefix-map=$(TOOLCHAIN_DIR)=xtensa-lx106-elf -gno-record-gcc-switches
 
-MFORCE32 := $(shell $(CC) --help=target | grep mforce-l32)
-ifneq ($(MFORCE32),)
-    # If the compiler supports the -mforce-l32 flag, the compiler will generate correct code for loading
-    # 16- and 8-bit constants from program memory. So in the code we can directly access the arrays
-    # placed into program memory.
-    CFLAGS +=  -mforce-l32
-else
-	# Otherwise we need to use a helper function to load 16- and 8-bit constants from program memory.
-    CFLAGS += -DWITH_PGM_READ_HELPER
-endif
-
```

Comment:
Can this be kept around for the folks who build their toolchain with the -mforce-l32 patch? IIRC this was the case in Sming project.

Reply:
Done, with an optional override macro in latest push.